src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -16,4 +18,14 @@ export default function Form526Entry({ location, children }) {
       </ITFWrapper>
     </EVSSClaimsGate>
   );
+
+  // wraps the app and redirects user if they are not enrolled
+  return (
+    <BetaApp
+      featureName={features.dashboard}
```

Comment:
Oh, just caught this. Should this be `features.allClaimes?`

Reply:
d'oh it was working locally because I'm opted into dashboard already. good catch